src/main.ts

```diff
@@ -7,12 +7,15 @@ import {
   SwaggerCustomOptions,
   SwaggerModule,
 } from '@nestjs/swagger';
+import { patchNestJsSwagger } from 'nestjs-zod';
 
 import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
 
+  patchNestJsSwagger();
+
   app.enableVersioning({
     type: VersioningType.URI,
   });
```

Comment:
This package has been deprecated.

https://github.com/risen228/nestjs-zod#readme

Reply:
removed the nestjs-zod package.